priv/repo/migrations/20250109154813_add_operation.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Wanda.Repo.Migrations.AddOperation do
+  use Ecto.Migration
+
+  def change do
+    create table(:operations, primary_key: false) do
+      add :operation_id, :uuid, primary_key: true
+      add :group_id, :uuid, null: false
+      add :result, :string, null: false
+      add :status, :string, null: false
+      add :targets, :map, null: false, default: "[]"
```

Comment:
Please could we use `:jsonb`? This is the correct migration type, the way we are using map works because postgres serialize everything under json and it "works" but the correct postgres type is the  jsonb one

This also for the agent_reports type
(I know we are using the same approach in the current migration for checks execution, I think in the future we should take care also of that part of migration)

Reply:
Also we can add a proper index to jsonb fields to speed up the queries (to both fields)

Reply:
Updated